self_enrollment_tool/templates/self_enrollment_tool/index.html

```diff
@@ -41,6 +61,7 @@ <h4 class="modal-title bottom-spacing">Self Enrollment Course List:</h4>
                     <th>Role&nbsp;ID</th>
                     <th>Self&nbsp;Enroll&nbsp;URL</th>
                     <th>Modified&nbsp;By</th>
+                    <th>Last&nbsp;Updated</th>
```

Comment:
Is this on qa? I don't see this column

Reply:
This was hidden and not used, it's now removed